vip-cache-manager/ttl-manager.php

```diff
@@ -27,6 +27,16 @@ function enforce_rest_api_read_ttl( $response, $rest_server, $request ) {
 		return $response;
 	}
 
+	// Don't override existing Cache-Control headers sent via PHP
+	$php_headers = headers_list();
+	foreach ( $php_headers as $header ) {
+		if ( 0 === stripos( $header, 'Cache-Control' ) ) {
```

Comment:
Should this be a stricter check?

```
if ( strtolower( $header ) === 'cache-control' ) {
```

That way we don't have false positive matches with things like `Cache-Control-Something-Else` (which could conceivably exist in the future at some point, or be a custom header from client).

Reply:
`$header` will look like `Cache-Control: max-age=60` so we can't do an exact check as suggested.

It's a good point though. `stripos` takes care of the case-insensitive matching for us already and to make it more specific, I'm now checking for `Cache-Control:` (with the colon).

Reply:
Ah ok, adding the colon will achieve the same. 👍